src/util.ts

```diff
@@ -82,14 +82,16 @@ export const execute = (cmd: string[]) => {
   core.debug(`Executing: \`${cmd.toString()}\``)
   // needs escaping if cmd[0] contains spaces
   // https://github.com/prefix-dev/setup-pixi/issues/184#issuecomment-2765724843
-  return exec(`"${cmd[0]}"`, cmd.slice(1))
+  return exec(`"${cmd[0]}"`, cmd.slice(1), { cwd: options.workingDirectory })
 }
 
-export const executeGetOutput = (cmd: string[], options?: ExecOptions) => {
+export const executeGetOutput = (cmd: string[], execOptions?: ExecOptions) => {
```

Comment:
we can remove the `?` here as there is only one usage of this function and that doesn't use `undefined`. then we also don't need the `execOptions ? { ...defaultOptions, ...execOptions } : defaultOptions` but can just use `{ ...defaultOptions, ...execOptions }`